store/snap_remote.go

```diff
@@ -86,7 +81,8 @@ func (s *RemoteSnap) Info() *snap.Info {
 		Revision:    s.Revision(),
 		Type:        s.Type(),
 		Channel:     s.Channel(),
-		Description: s.Description(),
+		Summary:     s.Pkg.Title,       // XXX: should be summary when the store provides it
+		Description: s.Pkg.Description, // XXX not quite right but ok for now
```

Comment:
well the first line afaiu is the tagline (store only concept), not the summary, so not sure it's a win atm